lms/djangoapps/django_comment_client/base/views.py

```diff
@@ -608,16 +610,6 @@ def follow_commentable(request, course_id, commentable_id):
 @require_POST
 @login_required
 @permitted
-def follow_user(request, course_id, followed_user_id):
```

Comment:
@explorerleslie Do you care if I remove these untested and unused methods that supposedly would allow "following" and "unfollowing" a user?